_config.yml

```diff
@@ -1,10 +1,14 @@
 title: JKAN
 description: A lightweight, backend-free open data portal, powered by Jekyll.
 baseurl: /jkan
+logo:
```

Comment:
Got it -- we should probably add a default logo then, so `_config.yml` can have a default value. Know any logo designers? :P